File: RecyclerveiwApp/app/src/main/java/com/example/recyclerveiwapp/MainActivity.java
package com.example.recyclerveiwapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {
    RecyclerView recyclerView;
    String[] arr = {"Item 1" , "Item 2" , "Item 4" ,  "Item 5" , "Item 6" ,"Item 7" ,"Item 8" ,"Item 10" };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        recyclerView = findViewById(R.id.recyclerView);
    }
}

File: DayamPortfolio2/app/src/main/java/com/example/dayamportfolio/MainActivity.kt
package com.example.dayamportfolio

import android.annotation.SuppressLint
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    @SuppressLint("MissingInflatedId")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val buttonSkills = findViewById<Button>(R.id.btnskills)
        buttonSkills.setOnClickListener {
            intent = Intent(this, SkillsActivity::class.java)
            startActivity(intent)
        }
        val buttonEducation = findViewById<Button>(R.id.btneducation)
        buttonEducation.setOnClickListener {
            intent = Intent(this, EducationActivity::class.java)
            startActivity(intent)
        }
        val work = findViewById<Button>(R.id.work)
        work.setOnClickListener {
            val workFragment = workFragment()
            supportFragmentManager.beginTransaction()
                .replace(R.id.container, workFragment)
                .addToBackStack(null)
                .commit()
        }
    }



}